homeassistant/components/rova/sensor.py

```diff
@@ -140,7 +141,9 @@ def update(self):
         self.data = {}
 
         for item in items:
-            date = datetime.strptime(item["Date"], "%Y-%m-%dT%H:%M:%S")
+            date = datetime.strptime(item["Date"], "%Y-%m-%dT%H:%M:%S").replace(
+                tzinfo=get_time_zone("Europe/Amsterdam")
```

Comment:
ROVA is a waste collection company that operates in the center and east of the Netherlands. 

https://www.home-assistant.io/integrations/rova

Reply:
Nice! Actually, being able to handle it like this is kinda nice